rasa/cli/arguments/default_arguments.py

```diff
@@ -27,6 +27,18 @@ def add_model_param(
         )
 
 
+def add_no_plot_param(
+    parser: argparse.ArgumentParser, default: bool = False, required: bool = False,
+):
+    parser.add_argument(
+        "--no-plot",
```

Comment:
More of a stylistic suggestion, but I prefer having verbs in the names (`should_do_x`, `disable_plotting`)
```suggestion
        "--disable-plotting",
```

Reply:
I agree, especially for naming of bool variables, but I hope I'm doing a right suggestion saying that we already have a certain way of names for cli like "--no-errors". Also, "--no-plot" is shorter and thus more grammar-error prone for the end users 🙂

Reply:
True, then let's just do the rename in the function parameters